spec/join.js

```diff
@@ -0,0 +1,7 @@
+module.exports = function (array) {
+  for (i = 0; i < array.length; i++) {
+
+
+  }
+  return array.filter(/\s/g,'~')
```

Comment:
wow you like regex 😁  ...I used to code in Perl...so I really like regex too. Again I would challenge yourself not to you use built in functions like ```filter()``` so you get more practice problem solving.

Just for this project! For everything else they are great!!! and encouraged to use!